src/site/stages/build/process-cms-exports/schemas/raw/node-outreach-asset.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
+const { getFilter } = require('../../filters');
+const nodeStandardKeys = require('../standard-keys/node');
+
+module.exports = {
+  type: 'object',
+  properties: {
+    // Standard `node` key-value pairs.
+    ...nodeStandardKeys,
```

Comment:
Abstracted standard `node` key-value pairs as per [the documentation](https://github.com/department-of-veterans-affairs/vets-website/pull/11041/files#diff-90a007c49873bf6e0d0d6eea7bffb437).